pkgs/build-support/fetchsteam/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  runCommand,
+  depotdownloader,
+  cacert,
+  writeText,
+}:
+
+{
+  name ? "${toString app}-${toString depot}-${toString manifest}-depot",
+  app,
+  depot,
+  manifest,
+  branch ? null,
+  language ? null,
+  lowViolence ? false,
+  fileList ? [ ],
+  fileListRegex ? false,
+  debug ? false,
+  hash ? lib.fakeHash,
+}:
+
+runCommand name
+  {
+    nativeBuildInputs = [ depotdownloader ];
+
+    env.SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
+
+    outputHash = hash;
+    outputHashAlgo = "sha256";
+    outputHashMode = "recursive";
+  }
```

it also doesn't force a sha256 hash
```suggestion
    outputHash = hash;
    outputHashMode = "recursive";
  }
  // lib.optionalAttrs (hash == "") { outputHashAlgo = "sha256"; })
```